cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,136 @@
+    cy.get('select[aria-label="rows per page"] option').each(($option) => {
+      const optionText = $option.text();
+      const optionValue = $option.val();
+      cy.get('select[aria-label="rows per page"]').select(optionText);
+      cy.get('select[aria-label="rows per page"]').should(
+        'have.value',
+        optionValue
+      );
+    });
+  });
+
+  it('should have the ability to add a new worker', () => {
+    cy.generateWorker().then((worker) => {
+      cy.findById('addNewRecordButton').click();
+      cy.findById('firstName').type(worker.firstName);
+      cy.findById('lastName').type(worker.lastName);
+      cy.findById('userEmail').type(worker.email);
+      cy.findById('age').type(worker.age);
+      cy.findById('salary').type(worker.salary);
+      cy.findById('department').type(worker.department);
+      cy.findById('submit').click();
+      cy.get('.rt-tr:not(.rt-tr.-padRow.-odd, .rt-tr.-padRow.-even)')
+        .last()
+        .find('.rt-td')
+        .then(($td) => {
+          cy.wrap($td.eq(0)).should('have.text', worker.firstName);
+          cy.wrap($td.eq(1)).should('have.text', worker.lastName);
+          cy.wrap($td.eq(2)).should('have.text', worker.age);
+          cy.wrap($td.eq(3)).should('have.text', worker.email);
+          cy.wrap($td.eq(4)).should('have.text', worker.salary);
+          cy.wrap($td.eq(5)).should('have.text', worker.department);
+        });
+    });
+  });
+
+  it('should have the ability to delete a worker', () => {
+    cy.addWorker(1);
+    cy.get('.rt-tr:not(.rt-tr.-padRow.-odd, .rt-tr.-padRow.-even)')
+      .its('length')
+      .then((countOfRows) => {
+        cy.get('[title="Delete"]').last().click();
+        cy.get('.rt-tr:not(.rt-tr.-padRow.-odd, .rt-tr.-padRow.-even)')
+          .its('length')
+          .should('eq', countOfRows - 1);
+      });
+  });
+
+  it('should have the ability to delete all workers', () => {
+    cy.get(
+      '[title="Delete"]:not(.rt-tr.-padRow.-odd, .rt-tr.-padRow.-even)'
+    ).then(($buttons) => {
+      const countOfRows = $buttons.length;
+
+      function deleteRow() {
+        cy.get(
+          '[title="Delete"]:not(.rt-tr.-padRow.-odd, .rt-tr.-padRow.-even)'
+        )
+          .first()
+          .click();
+      }
+      for (let i = 0; i < countOfRows; i++) {
+        deleteRow();
+      }
+      cy.contains('No rows found').should('exist');
+    });
+  });
+
+  it('should have the ability to find a worker in the search field and edit it', () => {
+    const editedFirstName = 'editTest';
+    cy.generateWorker().then((worker) => {
+      cy.findById('addNewRecordButton').click();
+      cy.findById('firstName').type(worker.firstName);
+      cy.findById('lastName').type(worker.lastName);
+      cy.findById('userEmail').type(worker.email);
+      cy.findById('age').type(worker.age);
+      cy.findById('salary').type(worker.salary);
+      cy.findById('department').type(worker.department);
+      cy.findById('submit').click();
```

Comment:
```suggestion
   cy.addWorker(1);
```
you have add worker function